source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.md

```diff
@@ -1258,12 +1260,11 @@ These options were originally only implemented in three drivers (Java, C#, and P
 these fields would allow for faster diagnosis of issues in the connection pool, they would not actually prevent an error
 from occurring.
 
-Additionally, these options have the effect of prioritizing older requests over newer requests, which is not necessarily
-the behavior that users want. They can also result in cases where queue access oscillates back and forth between full
-and not full. If a driver has a full waitQueue, then all requests for [Connections](#connection) will be rejected. If
-the client is continually spammed with requests, you could wind up with a scenario where as soon as the waitQueue is no
-longer full, it is immediately filled. It is not a favorable situation to be in, partially b/c it violates the fairness
-guarantee that the waitQueue normally provides.
+Additionally, these options have the effect of prioritizing newer requests over older requests, which is not necessarily
+the behavior that users want. For example, consider a situation when a waitQueue is full, and a request for
+[Connection](#connection) gets rejected. Then a spot opens in the waitQueue, and a newer request gets accepted. One may
+say that the newer request was prioritized over the older one, which violates the fairness recommendation that the
+waitQueue normally adheres to.
```

Comment:
Optional: Consider making the capitalization of WaitQueue in this paragraph consistent with (most of) the rest of the doc.

```suggestion
Additionally, these options have the effect of prioritizing newer requests over older requests, which is not necessarily
the behavior that users want. For example, consider a situation when a WaitQueue is full, and a request for
[Connection](#connection) gets rejected. Then a spot opens in the WaitQueue, and a newer request gets accepted. One may
say that the newer request was prioritized over the older one, which violates the fairness recommendation that the
WaitQueue normally adheres to.
```